polyfill/lib/monthday.mjs

```diff
@@ -8,15 +8,17 @@ export class MonthDay {
   constructor(month, day, disambiguation = 'constrain') {
     month = ES.ToInteger(month);
     day = ES.ToInteger(day);
+    disambiguation = ES.ToString(disambiguation);
+    const leapYear = 1972;
```

Comment:
Basically because I felt like we should at least be able to create a `MonthDay` for 29 February. Filed https://github.com/tc39/proposal-temporal/issues/261, though.